controlplane/pkg/apis/connectioncontext/helpers.go

```diff
@@ -45,3 +45,42 @@ func (c *ConnectionContext) MeetsRequirements(original *ConnectionContext) error
 
 	return nil
 }
+
+func (c *ExtraPrefixRequest) IsValid() error {
+	if c == nil {
+		return fmt.Errorf("ExtraPrefixRequest should not be nil...")
+	}
+
+	if c.RequiredNumber < 1 {
+		return fmt.Errorf("ExtraPrefixRequest.RequiredNumber should be positive number >=1: %v", c)
```

Comment:
No need to put the full `ExtraPrefixRequest.RequiredNumber` in this and consequent messages. Please replace with simple `RequiredNumber`. Please clean all the messages below too.

Reply:
This is done to be consistent with other helpers.go code.

https://github.com/ligato/networkservicemesh/blob/master/controlplane/pkg/apis/local/connection/helpers.go and others.